Friend/firmware/firmware_v1.0/README.rst

```diff
@@ -1,96 +1,63 @@
 
-The source code shows how to:
+## Directory Structure
 
-#. Get a pin specification from the :ref:`devicetree <dt-guide>` as a
-   :c:struct:`gpio_dt_spec`
-#. Configure the GPIO pin as an output
-#. Toggle the pin forever
+- `src/`: Contains the source code files
+- `include/`: Header files
+- `boards/`: Board-specific configurations (not used)
+- `overlays/`: Device-specific overlay files
+- `CMakeLists.txt`: CMake build configuration
+- `prj.conf`: Project configuration file
 
-See :zephyr:code-sample:`pwm-blinky` for a similar sample that uses the PWM API instead.
+## Prerequisites
 
-.. _blinky-sample-requirements:
+- (Visual Studio Code)[https://code.visualstudio.com/]
+- (nRF Connect for VS Code extension)[https://marketplace.visualstudio.com/items?itemName=NordicSemiconductor.nrf-connect-for-visual-studio-code]
+- (nRF Command Line Tools)[https://www.nordicsemi.com/Software-and-tools/Development-Tools/nRF-Command-Line-Tools/Download]
 
-Requirements
-************
+## Building the Firmware
 
-Your board must:
+1. Open the project in Visual Studio Code.
+2. Install the recommended VS Code extensions when prompted.
+3. Use the nRF Connect extension to build the firmware:
+   - Open the nRF Connect extension sidebar.
+   - Select your project configuration.
+   - Click on "Build" in the extension's toolbar.
 
-#. Have an LED connected via a GPIO pin (these are called "User LEDs" on many of
-   Zephyr's :ref:`boards`).
-#. Have the LED configured using the ``led0`` devicetree alias.
+## Flashing the Firmware
 
-Building and Running
-********************
+1. Double-tap the reset button on your device to enter bootloader mode. This will open a USB drive on your computer which is used for flashing.
+2. Locate the `zephyr.uf2` file in your build output directory.
+3. Copy the `zephyr.uf2` file to the USB drive that appeared when you put the device in bootloader mode.
+4. The device will automatically flash and restart with the new firmware.
 
-Build and flash Blinky as follows, changing ``reel_board`` for your board:
+## Device-Specific Builds
 
-.. zephyr-app-commands::
-   :zephyr-app: samples/basic/blinky
-   :board: reel_board
-   :goals: build flash
-   :compact:
+For different device versions (e.g., V1 and V2), use separate overlay files and project configuration files. See `CMakePresets.json` for the available configurations.
 
-After flashing, the LED starts to blink. If a runtime error occurs, the sample
-exits without printing to the console.
+These overlay files provide context on pins and device functions to the firmware when building. Each device will need its own unique build.
 
-Build errors
-************
+To select the appropriate overlay file select the configuration in the nRF Connect extension sidebar which is set in `CMakePresets.json`.
```

Comment:
Can't we aim to have a single firmware instead of using overlays during build ?
This will make things more confusing for users.
DevKit 2 is just an extension of DevKit1, if the pins are configured but not used, that shouldn't be an issue.